src/main/java/net/sf/jabref/gui/importer/fetcher/EntryFetcher.java

```diff
@@ -8,6 +8,7 @@
 import net.sf.jabref.logic.importer.OutputPrinter;
 
 /**
+ * <b>use SearchBasedFetcher instead</b> <br>
```

Comment:
Please use the [@deprecated](http://stackoverflow.com/a/24605999/873282) annotation.